### ISSUE TITLE
Request to include "VersionLabel"

### ISSUE THREAD

**Issue by not-ajayy** [aws-samples/sample-python-helper-aws-appconfig#17]
We need to print "VersionLabel" also in logs whenever latest configuration is fetched. 

In get_latest_configuration call we will get this "VersionLabel" which can be helped to print it.

Response Syntax
{
    'NextPollConfigurationToken': 'string',
    'NextPollIntervalInSeconds': 123,
    'ContentType': 'string',
    'Configuration': StreamingBody(),
    'VersionLabel': 'string'
}

Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/appconfigdata/client/get_latest_configuration.html

**Comment by jamesoff** [comment]
I'll add it as a property on the class that you can check - your code can print it out when it detects a change (that means you can print/format it in the way which works for you rather than being limited by whatever I choose).

**Comment by not-ajayy** [comment]
Hi @jamesoff  Can we get this released any time soon ?

**Comment by jamesoff** [comment]
I'm hoping to get it done this week for you :)

**Comment by not-ajayy** [comment]
Ok, will check it out, would help if its added in this module also.

**Comment by jamesoff** [comment]
I've published version 2.2.0 which adds the `version_label` property to the helper class.

Here's a quick snippet showing how you could use it:

```python
appconfig = AppConfigHelper(
    "MyAppConfigApp",
    "MyAppConfigEnvironment",
    "MyAppConfigProfile",
    45  # minimum interval between update checks
)

appconfig.update_config()
print(f"Version label is: {appconfig.version_label}")
```

**Comment by daninfpj** [comment]
Hello!

This change broke the code I was using:

```
File "/srv/app/.venv/lib/python3.13/site-packages/appconfig_helper/appconfig_helper.py", line 199, in update_config
    self._version_label = response["VersionLabel"]
                          ~~~~~~~~^^^^^^^^^^^^^^^^
KeyError: 'VersionLabel'
```

Does the response always include this key? Seems like in some cases it is not being included, although the documentation isn’t 100% clear about this. Can it be made optional?

**Comment by jamesoff** [comment]
@daninfpj sorry about that - I did some quick tests and it looked like it would be present but empty if not otherwise set. I've just updated and released 2.2.1 which should handle it better for you.

I'll also reach out to the service team to see if they can improve the documentation. (Edit: what is the source for your configuration? Is it something other than the AppConfig Hosted Config store?)

**Comment by daninfpj** [comment]
@jamesoff Thanks so much for the quick reply!
